src/global/tracking/tracking.cc

```diff
@@ -275,50 +277,208 @@ void InitPlugin(JApplication *app) {
+            "B0TrackerRawHitAssociations",
+        },
+        {
+            "B0TrackerCKFTruthSeededTrajectories",
+            "B0TrackerCKFTruthSeededTrackParameters",
+            "B0TrackerCKFTruthSeededTracks",
+            "B0TrackerCKFTruthSeededTrackAssociations",
+        },
+        app
+    ));
+
+    app->Add(new JOmniFactoryGeneratorT<TrackSeeding_factory>(
+        "B0TrackerTrackSeedingResults",
+        {
+            "B0TrackerRecHits"
+        },
+        {
+            "B0TrackerSeedingResults"
+        },
+        {},
+        app
+        ));
+
+    app->Add(new JOmniFactoryGeneratorT<CKFTracking_factory>(
+        "B0TrackerCKFTrajectories",
+        {
+            "B0TrackerSeedingResults",
+            "B0TrackerMeasurements"
+        },
+        {
+            "B0TrackerCKFActsTrajectoriesUnfiltered",
+            "B0TrackerCKFActsTracksUnfiltered",
+        },
+        app
+    ));
+
+    app->Add(new JOmniFactoryGeneratorT<ActsToTracks_factory>(
+        "B0TrackerCKFTracksUnfiltered",
+        {
+            "B0TrackerMeasurements",
+            "B0TrackerCKFActsTrajectoriesUnfiltered",
+            "B0TrackerRawHitAssociations",
+        },
+        {
+            "B0TrackerCKFTrajectoriesUnfiltered",
+            "B0TrackerCKFTrackParametersUnfiltered",
+            "B0TrackerCKFTracksUnfiltered",
+            "B0TrackerCKFTrackUnfilteredAssociations",
+        },
+        app
+    ));
+
+    app->Add(new JOmniFactoryGeneratorT<AmbiguitySolver_factory>(
+        "B0TrackerAmbiguityResolutionSolver",
+        {
+             "B0TrackerCKFActsTracksUnfiltered",
+             "B0TrackerMeasurements"
+        },
+        {
+             "B0TrackerCKFActsTracks",
+             "B0TrackerCKFActsTrajectories",
+        },
+        app
+    ));
+
+    app->Add(new JOmniFactoryGeneratorT<ActsToTracks_factory>(
+        "B0TrackerCKFTracks",
+        {
+            "B0TrackerMeasurements",
+            "B0TrackerCKFActsTrajectories",
+            "B0TrackerRawHitAssociations",
+        },
+        {
+            "B0TrackerCKFTrajectories",
+            "B0TrackerCKFTrackParameters",
+            "B0TrackerCKFTracks",
+            "B0TrackerCKFTrackAssociations",
+        },
+        app
+    ));
+
+
+    // COMBINED TRACKS
+
+    std::vector<std::string> input_track_collections, input_track_assoc_collections;
     //Check size of input_rec_collections to determine if CentralCKFTracks should be added to the input_track_collections
     if (input_rec_collections.size() > 0) {
         input_track_collections.push_back("CentralCKFTracks");
+        input_track_assoc_collections.push_back("CentralCKFTrackAssociations");
+    }
+    //Check if the B0Tracker readout is present in the current configuration
+    if (readouts.find("B0TrackerHits") != readouts.end()) {
+        input_track_collections.push_back("B0TrackerCKFTracks");
+        input_track_assoc_collections.push_back("B0TrackerCKFTrackAssociations");
```

Comment:
Since seeding doesn't work for B0 yet, people are fishing in 
`*TruthSeeded*` collection to get their B0 tracks.  We need to mix `B0TrackerCKFTruthSeededTracks` into `ReconstructedTruthSeededChargedWithoutPIDParticles` to not break current analyses.

Reply:
But that wasn't what was happening, right?

Reply:
Before this, B0 hits were mixed to both realistic and truth seeding. The proposed code only mixes in real seeded tracks (far backward doesn't ever use truth seeding).

Reply:
Ok, I see what you mean.

Reply:
20c61be317d1c900a20af9ed45343ff077a73c54